keyboards/handwired/ms_sculpt_mobile/teensypp/teensypp.c

```diff
@@ -0,0 +1 @@
+#include "teensypp.h"
```

Comment:
This file needs a GPLv2-compatible license header.

* [PR Checklist - Example GPLv2 Header - QMK Firmware Documentation](https://docs.qmk.fm/#/pr_checklist?id=example-gplv2-header)

Reply:
I thought at one point the style was to not require license for header files of less than 4 lines. Adding.